src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,29 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(4, auto);
+  justify-content: center;
+  gap: 48px;
+
+  @media (max-width: 1024px) {
```

Comment:
There should be four columns starting at 1024px. so it needs to be fixed
```suggestion
  @media (max-width: 1023px) {
```

Reply:
![image](https://user-images.githubusercontent.com/126285904/231156320-7f5deeb0-e9f2-438e-9998-4c58949fb418.png)
done